R-packages/evalcast/R/err_measures.R

```diff
@@ -104,6 +99,35 @@ interval_coverage <- function(coverage) {
   }
 }
 
-
+#' Common parameter checks for score functions
+#'
+#' A set of common checks for score functions, meant to identify common causes
+#' of issues.
+#'
+#' @param quantiles vector of forecasted quantiles
+#' @param values vector of forecasted values
+#' @param actual_value actual_value, either as a scalar or a vector of the same 
+#'   length as `quantiles` and `values`
+#' @param id string to identify the caller of the function and displayed in
+#'   error messages (recommended to be the parent function's name)
+score_func_param_checker <- function(quantiles, values, actual_value, id = ""){
+  id_str = paste0(id, ": ")
+  if(length(actual_value) > 1){
```

Comment:
Are there cases where having a vector of actual values the same length as quantiles would be encountered?

Reply:
yeah, depending on the score card, it may be replicated.